types/recharts/index.d.ts

```diff
@@ -375,7 +371,7 @@ export interface LegendProps {
     onClick?: RechartsFunction;
     onMouseEnter?: RechartsFunction;
     onMouseLeave?: RechartsFunction;
-    onBBoxUpdate?: (box: { width: number; height: number; }) => void;
+    onBBoxUpdate?: RechartsFunction;
```

Comment:
hm, I'm not sure about this one. Is this one also overloaded? In the source code its either a box or null -> https://github.com/recharts/recharts/blob/9d42367e658cab6bed0a8f2517f17be9e0cf0957/src/component/Legend.js on line 150 resp. 160

Reply:
Good catch! My idea is that, whatever method argument we have, as long as the return type is `void` we could use `RechartsFunction`. In its current form, `dtslint` says an inline type definition is not the way the go, so I needed to change this line anyway.